docs/pages/eas/workflows/syntax.mdx

```diff
@@ -610,15 +632,34 @@ This job outputs the following properties:
 Runs [Maestro](https://maestro.mobile.dev/) tests on a build.
 
 ```yaml
-maestro:
-  # @info #
-  type: maestro
-  # @end #
-  environment: production | preview | development # optional, defaults to preview
-  image: string # optional, defaults to 'default'. See https://docs.expo.dev/build-reference/infrastructure/ for a list of available images.
-  params:
-    build_id: string # required
-    flow_path: string | string[] # required
+jobs:
+  my_job:
+    # @info #
+    type: maestro
+    # @end #
+    environment: production | preview | development # optional, defaults to preview
+    image: string # optional, defaults to 'default'. See https://docs.expo.dev/build-reference/infrastructure/ for a list of available images.
+    params:
+      build_id: string # required
+      flow_path: string | string[] # required
+      shards: number # optional, defaults to 1
+      retries: number # optional, defaults to 1
+```
+
+#### `slack`
+
+Sends a message to a Slack channel using a webhook URL.
+
+```yaml
+jobs:
+  my_job:
+    # @info #
+    type: slack
+    # @end #
+    params:
+      webhook_url: string # required
+      message: string # required if payload is not provided
+      payload: object # required if message is not provided
```

Comment:
Could you please add an example of how one would do it? I tested it with
```yaml
jobs:
  slack_notification:
    type: slack
    params:
      message: "hello from workflow"
      webhook_url: "https://hooks.slack.com/services/[...]"
```

Might be interesting to give people a smart example which ties together some jobs and then sends a Slack notification in an `after` job reporting status of a workflow or a job.

Reply:
i have plans to add all prepackaged jobs to a new doc that has examples for each. for now im going ot leave this as is